data/transactions/logic/sourcemap.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2019-2022 Algorand, Inc.
```

Comment:
I also thought of putting this file in the `tealdbg` package, but I don't think it can be exported from there.